refactor: updated blue ring color to reach contrast ratio compliance (#1868)
Frontify/fondue: .changeset/weak-hounds-burn.md

---
"@frontify/fondue-components": patch
"@frontify/fondue": patch
---

> Updating the Ring Color to be compliant with a contrast ratio of 3:1

Reviewer: ```suggestion
refactor: update the outline color for better contrast ratio (3:1)
```
